pasystem/pasystem-impl/impl/src/java/org/sakaiproject/pasystem/impl/rest/PASystemEntityProvider.java

```diff
@@ -183,6 +183,11 @@ public String getTimezoneToolUrlForUser() {
             User thisUser = UserDirectoryService.getCurrentUser();
             String userid = thisUser.getId();
 
+            // If there is no user (e.g. on the gateway site!) there's no timezone
+            if(userid == null || userid == "") {
```

Comment:
Could this please use StringUtils.isBlank/isEmpty?
Also, why would there be a null/blank id returned? Shouldn't there just be no user?